### PR TITLE
doc improvement: misc doc edits for 1.7.0

### PR DESCRIPTION
**Pull request by greg-fer**
Added minor edits to several pages for NCS 1.7.0.
NCSDK-11020.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

----

The following tasks were created for more global doc improvements:

- [x] https://projecttools.nordicsemi.no/jira/browse/NCSDK-11184
- [x] https://projecttools.nordicsemi.no/jira/browse/NCSDK-11185
- [x] https://projecttools.nordicsemi.no/jira/browse/NCSDK-11198
- [x] https://projecttools.nordicsemi.no/jira/browse/NCSDK-11230

### CODE REVIEW

File: doc/nrf/ug_thread_tools.rst [hunk 1]
```diff
@@ -165,20 +165,20 @@ This option provides most of the functionalities available in the OpenThread Bor
 However, this approach requires you to download the OpenThread Border Router repository and install the Border Router manually on the Raspberry Pi.
 
 To set up and configure the OpenThread Border Router, follow the official `OpenThread Border Router Codelab tutorial`_ on the OpenThread documentation portal.
-After cloning the repository please ensure to checkout to the compatible commit id:
+After cloning the repository, ensure to check out the compatible commit id:
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_tools.rst:168]:
```suggestion
After cloning the repository, make sure to check out the compatible commit id:
```

**Comment by Avalei** [comment]
glossary.rst
- change "nRF Connect for Cloud" to "nRF Cloud".

getting-started.rst
- change "If this is the first time **you work** with embedded devices..." to "If this is the first time you **are working** with embedded devices..."

gs_assistant.rst
- Could we remove the note at the top mentioning GSA and west? It is stated above the GSA section and makes more sense to be located right by the title. It feel confusing to have it also located above the TCM section.

gs_installing.rst
- Under step 1 > the GN installation section > Windows tab > Step 3 - change "Ensure that the GN tool..." to "Check that the GN tool..."

gs_testing.rst
- "application documentation" should be possessive >> "application's documentation" in the following sentences:
  - Change "Follow the instructions in the testing section of the application documentation to ensure that the application runs as expected."
  - See the user interface section of the application documentation for description of the LED states or available UART commands.


I will continue reviewing the docs starting from "Updating tools and repos" on Monday next week.

**Comment by greg-fer** [comment]
> I will continue reviewing the docs starting from "Updating tools and repos" on Monday next week.

@Avalei , I've reviewed from bottom going up, up to Scripts (excluded). Also created two tasks for more global edits (see description).

**Review by hakonfam — APPROVED** [review]
SPM readme and bootloader docs are OK

File: samples/nrf9160/modem_shell/README.rst [hunk 2]
```diff
@@ -29,89 +30,118 @@ You can store some link subcommand parameters into settings, which are persisten
...
 --------
 
-Send AT command to query network status with::
+* Send AT command to query network status with:
+
+  .. code-block:: console
+
+     at at+cereg?
+
+* Send AT command to query neightbour cells:
```
Comment (umapraseeda) [inline samples/nrf9160/modem_shell/README.rst:97]:
```suggestion
* Send AT command to query neighbor cells:
```

File: doc/nrf/ug_bootloader_external_flash.rst [hunk 3]
```diff
@@ -12,7 +12,7 @@ See :ref:`pm_external_flash` for general information about how to set up partiti
 
 .. note::
 
-   Currently, only MCUboot supports booting images stored in the external memory partition.
+    Currently, only MCUboot supports booting images stored in the external memory partition.
```
Comment (wiba-nordic) [inline doc/nrf/ug_bootloader_external_flash.rst:15]: Why was a space added here? Isn't three spaces the correct amount for a note?

**Comment by wiba-nordic** [comment]
`doc_build.rst`:144-150 : These lines use the term environment variable. Does the variable need the new :envvar: tag, or should the term be changed?

**Review by umapraseeda — APPROVED** [review]

File: doc/nrf/ug_bootloader_external_flash.rst (hunk 3 above)
Comment (greg-fer) [reply doc/nrf/ug_bootloader_external_flash.rst:15]: Out of habit, I guess. I always indented notes compared with the tag (and rarely or never used the blank line after tag). But I see these notes display well, so I revert.

**Review by wiba-nordic — APPROVED** [review]

**Pull request merged**